src/Automation/Data/ScanOptions.cs

```diff
@@ -14,6 +14,11 @@ public class ScanOptions
         /// </summary>
         public string ScanId { get; }
 
+        /// <summary>
+        /// The window handle for the root of the UIA subtree to scan.
+        /// </summary>
+        public System.IntPtr WindowHandle { get; set; } = System.IntPtr.Zero;
```

Comment:
`WindowHandle` seems too vague here. Maybe `ScanRootWindowHandle` would be more descriptive.

Reply:
Agreed.  Updating.

Reply:
Added in latest iteration